waspc/src/Wasp/Generator/ExternalConfig/TsConfig.hs

```diff
@@ -0,0 +1,56 @@
+{-# LANGUAGE FlexibleInstances #-}
+
+module Wasp.Generator.ExternalConfig.TsConfig
+  ( validateTsConfig,
+  )
+where
+
+import qualified Wasp.ExternalConfig.TsConfig as T
+
+validateTsConfig :: T.TsConfig -> IO [String]
```

Comment:
Same thing like for validatePackageJson, if I am correct this doesn't have to be `IO`.